producer-c/producer-cloudwatch-integ/jobs/runner.groovy

```diff
@@ -0,0 +1,251 @@
+import jenkins.model.*
+import org.jenkinsci.plugins.workflow.steps.FlowInterruptedException
+
+START_TIMESTAMP = new Date().getTime()
+HAS_ERROR = false
+
+RUNNING_NODES=0
+
+def buildProducer() {
+  sh  """
+    cd $WORKSPACE/producer-c/producer-cloudwatch-integ && 
+    if [ ! -d "build" ] 
```

Comment:
You can shorten this to just `mkdir -p build`. The `-p` will not create a new directory if it exists already.